c2corg_ui/templates/helpers/edit.html

```diff
@@ -20,9 +20,11 @@
   </section>
 </%def>
 
-<%def name="show_editing_associated_routes(model)">\
+<%def name="show_editing_associated_routes(model, show_title = True)">\
```

Comment:
It's more consistent that default values are ``False`` => use ``hide_title`` instead?